cli/src/main/java/com/devonfw/tools/ide/context/AbstractIdeContext.java

```diff
@@ -178,6 +178,11 @@ public AbstractIdeContext(IdeLogLevel minLogLevel, Function<IdeLogLevel, IdeSubL
       currentDir = getParentPath(currentDir);
     }
 
+    // fallback to main
+    if (workspace.isEmpty()) {
+      workspace = WORKSPACE_MAIN;
+    }
+
```

Comment:
BTW: I would be a little bit more clean to solve this in line 168 like this:
```java
        if (FOLDER_WORKSPACES.equals(name1) && !name2.isEmpty()) {
```

Reply:
I just edited this online on github. Seems cleaner to not set workspace to empty string and then reset to `main` afterwards. So we can merge and include in todays release.